proxy/core/connection/server.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, host: str, port: int) -> None:
         self._conn: Optional[Union[ssl.SSLSocket, socket.socket]] = None
         self.addr: Tuple[str, int] = (host, int(port))
         self.closed = True
+        self.handshake: Dict[int, bytes] = {}
+        self.session_id: Optional[bytes] = None
```

Comment:
I am a little confused right now.  How is connection pooling even working currently.  As in who is the provider of `handshake` and `session_id` parameters.  Are we missing any file?

Reply:
These should be removed too